docs/content/connectors/db2-cdc.md

```diff
@@ -246,8 +246,7 @@ Data Type Mapping
 <table class="colwidths-auto docutils">
     <thead>
       <tr>
-        <th class="text-left" style="width:30%;">Db2 type<a href="https://dev.mysql.com/doc/man/8.0/en/data-types.
-html"></a></th>
+        <th class="text-left" style="width:30%;"><a href="https://www.ibm.com/docs/en/db2-for-zos/12?topic=elements-data-types">Db2 type</a></th>
```

Comment:
I think this is better to link to https://www.ibm.com/docs/en/db2/11.5?topic=elements-data-types.

Reply:
Thanks @ruanhang1993 for this suggestion.
Fixed.